extension/index.html

```diff
@@ -1,85 +1,94 @@
-        <b>general accessibility</b>.
-      </div>
-
-      <div class="changes-saving-note">
-        Note that in the form below your changes will save automatically:
-      </div>
-      <div class="sub-title">What do you identify with?</div>
-
-      <div class="description">
-        <form>
-          <label for="add">
-            <input type="checkbox" id="add" name="add" />
-            Attention Deficit Disorder </label
-          ><br />
-
-          <label for="blind">
-            <input type="checkbox" id="blind" name="blind" />
-            Blind </label
-          ><br />
-
-          <label for="cognitive">
-            <input type="checkbox" id="cognitive" name="cognitive" />
-            Cognitive </label
-          ><br />
-
-          <label for="colorblind">
-            <input type="checkbox" id="colorblind" name="colorblind" />
-            Color Blind </label
-          ><br />
-
-          <label for="deaf">
-            <input type="checkbox" id="deaf" name="deaf" />
-            Deaf </label
-          ><br />
-
-          <label for="lowvision">
-            <input type="checkbox" id="lowvision" name="lowvision" />
-            Low Vision
-          </label>
-          <br />
-
-          <label for="motor">
-            <input type="checkbox" id="motor" name="motor" />
-            Motor Function Disability </label
-          ><br />
-
-          <label for="sightedkeyboarduser">
-            <input
-              type="checkbox"
-              id="sightedkeyboarduser"
-              name="sightedkeyboarduser"
-            />
-            Sighted Keyboard User
-          </label>
-          <br />
-        </form>
-      </div>
-    </section>
-  </body>
-</html>
+
+<head>
+  <title>Ha11y</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+  <link rel="stylesheet" href="styles.css" />
+  <script src="options.js"></script>
+</head>
+
+<body>
+  <header>
+    <div class="logo">
+      <img src="../icon.png" alt="Ha11y Logo" width="100px" height="100px" />
+    </div>
+    <h1>Ha11y</h1>
+  </header>
+
+  <main>
+    <h2>What is Ha11y?</h2>
+    <p>
+      Ha11y allows users to discover whether or not a website will be
+      <b>accessible</b> to them before even opening the website. Ha11y Google
+      Chrome Extension scores and displays the accessibility score of websites
+      directly in a google search
+      <b>tailored directly around the users</b> disability. When users first
+      signup, they can select their disability and see personally-tailored
+      results; otherwise, there is a default score for
+      <b>general accessibility</b>.
+    </p>
+
+    <h3>
+      Note that in the form below your changes will save automatically:
+    </h3>
+    <h2>Settings:</h2>
+    <form>
+      <p>What do you identify with?</p>
```

Comment:
can you make the questions a bit bigger?

Reply:
there is also more space after the question than before and that makes the grouping a bit odd